### ISSUE TITLE
Fair data types

### ISSUE THREAD

**Issue by svenvanderburg** [fair-workflows/fairworkflows#123]
* [x] Each input/output variable is represented by a URI, there is a RDF:type triple with an object that is a string literal denoting the type of the input.
* [x] When creating a fairstep from a function we should parse the type from the type annotation of the function and store it in the RDF.
In principle these should be semantic classes instead of string literals, but the exact form of these and the definition of comparison still need to be provided.

RDF will be something like:
```
    step 
          has input uri1
          has output uri2
     uri1 isof type 'string'
     uri2 isoftype 'int'
```

**Comment by svenvanderburg** [comment]
@raar1 maybe [xml data type ontology](https://www.w3.org/TR/2004/REC-xmlschema-2-20041028/) could be used.

**Comment by svenvanderburg** [comment]
Fixed by #129
